api/const.go

```diff
@@ -0,0 +1,4 @@
+package api
```

Comment:
Isn't it cleaner to have `ZONE_SBS` in `api.go`, and `ENVIRONMENT_P` in `api_test.go`? Since that's where they are used.